### DISCUSSION TITLE
How to migrate Skia.WPF project from Uno 4 to Uno 5?

### DISCUSSION THREAD

**Discussion by LITTOMA** [unoplatform/uno#14281] [Q&A] [answered]
### What would you like clarification on

I changed the initialization of WpfHost to:
`root.Content = new global::Uno.UI.Runtime.Skia.Wpf.WpfHost(Dispatcher, () => new AppHead());`

but it just shows a string 'Uno.UI.Runtime.Skia.Wpf.WpfHost' in the window.
![image](https://github.com/unoplatform/uno/assets/10324313/4b127d64-94fa-4822-b7f0-6497a84c0cab)

What are the breaking changes?

### Concern

None

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question! We've documented the migration here: https://platform.uno/docs/articles/migrating-from-previous-releases.html
